CHANGELOG.md

```diff
@@ -1,7 +1,21 @@
+## 1.12.1 - ???
+
+- Add support for working_dates (an option to mark some weekend as a "working day")
```

Comment:
It might be clearer to describe this as:

> Add support for specifying `working_dates` (special dates that are "working days", even though they are not one of the specified working days, e.g. weekend dates that are special working days)

Reply:
It should also say `extra_working_dates`.